Terminal.Gui/Core/PosDim.cs

```diff
@@ -499,7 +499,7 @@ public static Dim Sized (int n)
 			return new DimAbsolute (n);
 		}
 
-		class DimCombine : Dim {
+		public class DimCombine : Dim {
```

Comment:
If it's public it needs documentation.

Reply:
Sorry. There no need to be public. Changing to `internal `like the `DimAbsolute `is enough. Thanks.